starknet_py/contract.py

```diff
@@ -270,13 +282,18 @@ def prepare(
         :return: PreparedFunctionCall
         """
         if version is None:
+            # version = (
+            #     self._client.supported_tx_version
+            #     if isinstance(self._client, AccountClient)
+            #     else 0
+            # )
+            # TODO add better logic without using internal account
```

Comment:
Is it WIP or something else?

Reply:
As you can see by `TODO` 😅. I forgot about this one, will be changed.

Reply:
Actually, this is final. I've removed commented out code and TODOs.